src/test/java/edu/ucsb/cs156/example/controllers/CSRFControllerTests.java

```diff
@@ -23,8 +25,17 @@
 
   @Test
   public void csrf_returns_ok() throws Exception {
-      mockMvc.perform(get("/csrf"))
-              .andExpect(status().isOk());
+    MvcResult response = mockMvc.perform(get("/csrf"))
+              .andExpect(status().isOk())
+              .andReturn();
+
+    String responseString = response.getResponse().getContentAsString();
+    assertTrue(!responseString.equals(""));
```

Comment:
Isn't this assertion redundant due to the below assertions?

Reply:
Yes, good point.  I'll delete that line.

Reply:
Done